checkov/sca_image/runner.py

```diff
@@ -52,8 +62,8 @@ def scan(
             image_scanner.cleanup_scan()
             raise
 
-    @staticmethod
     async def execute_scan(
```

Comment:
why did you choose to put the method "execute_scan" in the runner rather than in "image_scanner.py"?

Reply:
It was originally in the runner, I haven't moved it.
Should I move it to the image_scanner?